src/ui/viewmodels/MemoryViewerViewModel.hh

```diff
@@ -175,6 +175,10 @@ public:
     void AdvanceCursorPage();
     void RetreatCursorPage();
 
+    void SaveToMemViewHistory();
```

Comment:
Suggest `AddToHistory`, saving implies something that gets remembered between sessions.

Might also want a `ClearHistory` method so the addresses can be reset when changing games.